ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/NewMigrationCommand.java

```diff
@@ -92,6 +116,16 @@ private boolean tryCreatePropertiesFile(final String path) {
       LOGGER.error(String.format("Failed to create file %s: %s", path, e.getMessage()));
       return false;
     }
+
+    final String initialConfig = MigrationConfig.KSQL_SERVER_URL + "=" + ksqlServerUrl;
```

Comment:
How will we add more default lines to the config file in the future?

Reply:
Not sure I understand your question -- any extra lines we add to the default config should also be added here (in the same place). We can pull this out into a helper method and switch to using StringBuilder when the time comes.

Reply:
Was asking since this section could get long if we include comments and TLS configs. What you suggested works though.